telecine/src/main/java/com/jakewharton/telecine/TelecineService.java

```diff
@@ -40,10 +48,37 @@ static Intent newIntent(Context context, int resultCode, Intent data) {
   private RecordingSession recordingSession;
 
   private final RecordingSession.Listener listener = new RecordingSession.Listener() {
+    private boolean showTouches;
+    private boolean useDemoMode;
```

Comment:
kind of unrelated, but, previously, if you toggle "show touches" off during a recording, `onStop()` never turns off the touches.